### ISSUE TITLE
Avis home page! Close button not labeled

### ISSUE THREAD

**Issue by Chetna1307** [Kishan1601/Test-1#3]
Environment
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5

Repro Steps
1. Open the URL https://www.avis.com/en/home
2. Press Tab Key to navigate to the Close button next to Travel Safely and Flexibly option

Expected Result
the screen reader announce the close button when in focus 

Actual Result
the button to close the upper banner is announced as “button” by the screen reader

WCAG Guideline
https://www.w3.org/TR/WCAG21/#name-role-value
